package.json

```diff
@@ -31,13 +31,15 @@
 		"chai-http": "*",
 		"nodemon": "*",
 		"apidoc": "*",
-		"mocha": "*"
+		"mocha": "*",
+		"eslint": "^5.15.0"
 	},
 	"scripts": {
 		"test": "mocha api/test/",
 		"docs": "apidoc -i api/controller -i dashboard/helper -o docs/www/",
 		"start": "node sugarizer.js",
 		"start:watch": "nodemon sugarizer.js",
-		"seed:users-csv": "node scripts/seed_users.js"
+		"seed:users-csv": "node scripts/seed_users.js",
+		"lint": "eslint .; exit 0"
```

Comment:
why do we need `exit 0`

Reply:
If ESlint detects errors in the file, it exits with error code 1. The output looks like the node script is crashing. I've used `exit 0` to suppress the error.

Reply:
yeah you can format this file https://github.com/llaske/sugarizer-server/blob/master/dashboard/public/js/main.js and move code to respective controllers.